src/main/java/net/neoforged/neoforge/common/IShearable.java

```diff
@@ -42,20 +49,55 @@ default boolean isShearable(ItemStack item, Level level, BlockPos pos) {
      * The object should perform all actions related to being sheared,
      * except for dropping of the items, and removal of the block.
      * As those are handled by ItemShears itself.
-     *
-     * Returns a list of items that resulted from the shearing process.
-     *
+     * <p>
      * For entities, they should trust there internal location information
```

Comment:
While you're at it, can you please turn this into a their or otherwise remove it as it doesn't really make a lot of sense.